gordo_components/model/anomaly/diff.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+
+import os
+import json
+
+import numpy as np
+import pandas as pd
+
+from typing import Optional, Union
+from datetime import timedelta
+
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.base import BaseEstimator, TransformerMixin
+
+from gordo_components import serializer
+from gordo_components.model.base import GordoBase
+from gordo_components.server.views import common
+from gordo_components.model.models import KerasAutoEncoder
+
+
+class DiffBasedAnomalyDetector(BaseEstimator, GordoBase):
+    def __init__(
+        self,
+        base_estimator: BaseEstimator = KerasAutoEncoder(kind="feedforward_hourglass"),
+        scaler: TransformerMixin = MinMaxScaler(),
```

Comment:
Should we use something else than MinMaxScaler as default? I don't have a competent opinion really, except the observation that MinMaxScaler is quite sensitive to even a single outlier. Maybe RobustScaler, which seems to be like MinMax, only it does not use the biggest/lowest, but the 25th and 75th quantile?